front/lib/api/assistant/agent_usage.ts

```diff
@@ -198,6 +198,9 @@ export async function agentMentionsUserCount(
             as: "userMessage",
             attributes: [], // No attributes are necessary here for grouping
             required: true,
+            where: {
+              userContextOrigin: "web",
```

Comment:
We likely need an index

If you want slack as well you can search for web or slack

Reply:
Slack includes workflow I guess? In the positive we probably only want to keep web mentions. 

If we take the example of our workspace and include workflows then we end up having the signupRadar assistant as the most popular one (which is not used by anyone)

Reply:
Slack includes workflow I guess? In the positive we probably only want to keep web mentions. 

If we take the example of our workspace and include workflows then we end up having the signupRadar assistant as the most popular one (which is not used by anyone)